app/views/components/docs/reorderable_list.yml

```diff
@@ -0,0 +1,54 @@
+name: Reorderable list
+description: A list of items that can be reordered
+body: |
+  List items can be reordered by drag and drop or by using the up/down buttons.
+  On small viewports the drag and drop feature is disabled to prevent being triggered
+  when scrolling on touch devices.
+
+  This component uses SortableJS - a JavaScript library for drag and drop interactions.
+  When JavaScript is disabled a set of inputs will be shown allowing users to provide
+  an order index for each item.
+accessibility_criteria: |
+  Buttons in this component must:
+
+  * be keyboard focusable
+  * inform the user about which item they operate on
+  * preserve focus after interacting with them
+examples:
+  default:
+    data:
+      items:
+        - title: "Budget 2018"
+          description: "PDF, 2.56MB, 48 pages"
+        - title: "Budget 2018 (web)"
+          description: "HTML attachment"
+        - title: "Impact on households: distributional analysis to accompany Budget 2018"
+          description: "PDF, 592KB, 48 pages"
+        - title: "Table 2.1: Budget 2018 policy decisions"
+          description: "MS Excel Spreadsheet, 248KB"
+        - title: "Table 2.2: Measures announced at Autumn Budget 2017 or earlier that will take effect from November 2018 or later (£ million)"
+          description: "MS Excel Spreadsheet, 248KB"
+  within_form:
+    embed: |
+      <form>
+        <%= component %>
+        <button class="govuk-button" type="submit">Save order</button>
+      </form>
+    data:
+      order_input_name: new_order
+      items:
+        - title: "Budget 2018"
+          description: "PDF, 2.56MB, 48 pages"
```

Comment:
I've talked to Simon that doing these trio of attributes is not going to be very easy as this is data that is wrapped up in the attachment component. So we talked about simplifying them. It shouldn't matter to you much since we're just passing a string in here though.